airflow/models/taskinstance.py

```diff
@@ -1199,23 +1199,22 @@ def __repr__(self):
 
         return {
             'dag': task.dag,
+            'dag_run': dag_run,
+            'run_id': run_id,
```

Comment:
And what do you think this change. At first I think template messy and all date variable should keep together, but I find out your PR like to keep them in alphabetical
Should I change them in alphabetical or keep them?

Reply:
Let's reorder alphabetically and then merge 👍